ejb3/src/main/resources/subsystem-templates/ejb3.xml

```diff
@@ -41,6 +41,12 @@
                <keepalive-time time="60" unit="seconds"/>
            </thread-pool>
        </thread-pools>
+       <server-interceptors>
+           <interceptor module="foo:interceptors:1.0" class="org.bar.FooInterceptor"/>
+       </server-interceptors>
+       <client-interceptors>
+           <interceptor module="foo:interceptors:1.0" class="org.bar.FooInterceptor"/>
+       </client-interceptors>
```

Comment:
@tadamski This must be removed. This isn't documentation, it must be able to generate the same config that we ship in standalone[-xxx].xml, which must be runnable.

Reply:
fixed